backend/README.md

```diff
@@ -1,6 +1,11 @@
 This directory contains code for the components that comprise the Kubeflow
 Pipelines backend.
 
+
+## Bazel
+
+There are some [know issues](https://github.com/kubeflow/pipelines/issues/3942) with the latest version of Bazel and the kubeflow pipelines repo, therefore use version 0.24.1 or older. 
```

Comment:
https://github.com/kubeflow/pipelines/search?q=bazel&unscoped_q=bazel

I think we are always using bazel 0.24.0 instead